src/DssSpell.t.sol

```diff
@@ -18,36 +18,28 @@ pragma solidity 0.6.12;
 
 import "./DssSpell.t.base.sol";
 
-interface RwaLiquidationLike {
-    function ilks(bytes32) external returns (string memory, address, uint48 toc, uint48 tau);
-    function bump(bytes32 ilk, uint256 val) external;
-    function tell(bytes32) external;
-    function cure(bytes32) external;
-    function cull(bytes32, address) external;
-    function good(bytes32) external view returns (bool);
+interface ERC20Like {
+    function balanceOf(address) external view returns (uint256);
 }
 
-interface RwaUrnLike {
-    function can(address) external view returns (uint256);
-    function lock(uint256) external;
-    function draw(uint256) external;
-    function wipe(uint256) external;
-    function free(uint256) external;
-}
+contract DssSpellTest is DssSpellTestBase {
 
-interface RwaOutputConduitLike {
-    function can(address) external view returns (uint256);
-    function may(address) external view returns (uint256);
-    function pick(address) external;
-    function push() external;
-}
+    function test_RWA009Draw() public {
 
-interface RwaInputConduitLike {
-    function may(address) external view returns (uint256);
-    function push() external;
-}
+        address dai         = addr.addr("MCD_DAI");
+        address conduit     = addr.addr("RWA009_A_OUTPUT_CONDUIT");
+        uint256 prevBalance = ERC20Like(dai).balanceOf(conduit);
```

Comment:
Not sure we can rely on conduit (Genesis address) balance here, they can move DAI at any time.

Reply:
Thanks for taking a look. I don’t think that’s an issue because the tests run in an isolated environment. It’s a mainnet fork where we deploy the spell and perform each of the tests. So there’s no way for them to move their funds in that context. Let me know if you were thinking about something else or if you have any questions.

Reply:
Hmmm but isn't that what Julien is solving by checking the difference between prev and post balance?

Reply:
Sorry I hadn't refreshed with your response Julien.